### DISCUSSION TITLE
Is there a way to disable namespaces in a page-explorer?

### DISCUSSION THREAD

**Discussion by biffhero** [combostrap/combo#27] [Q&A]
I have a "Sidebar" set up with the following:

<page-explorer tree ></page-explorer>

Is there a way to eliminate certain namespaces?  I tried this but it doesn't work.

<page-explorer tree ns-exclude:wiki:></page-explorer>

I'm hoping that the ns-exclude rule works here, or there's something else that I can do to make "PlayGround" and "Wiki" go away.

Thanks,
Rob

**Comment by biffhero** [comment]
It seems that github discussions doesn't like dokuwiki code.

page-explorer tree  <--- I have this, and I tried this ---> page-explorer tree ns-exclude:wiki:

Thanks,
Rob

**Reply by gerardnico** [reply]
Page explorer should be influenced by

https://www.dokuwiki.org/config:hidepages

**Reply by biffhero** [reply]
You are right, thank you.  I had used that page before, but when I did that, I still saw the folded namespaces, so I figured it didn't work.

Now, when I re-use those examples, the folded namespace name is there, but the underlying pages are not.  I presume this is as designed, as the variable is "hidepages", and not "hidenamespaces".

Thanks,
Rob

**Reply by gerardnico** [reply]
Not really sure. 

The page at the root tends to not have any sidebar because they are landing page.

**Reply by biffhero** [reply]
It's all good, I used 'mv', and those don't show up anymore.  Thank you for your help.

**Reply by gerardnico** [reply]
What do you mean by 'i use mv' ?
